Engines/Wine/Engine/Object/script.js

```diff
@@ -13,7 +13,7 @@ const ProcessBuilderClass = Java.type("java.lang.ProcessBuilder");
 // eslint-disable-next-line no-unused-vars
 module.default = class Wine {
     constructor() {
-        this._implementation = new WineEngine();
+                 this._implementation = new WineEngine();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 17.](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4786217)